pkg/processor/endpoints.go

```diff
@@ -0,0 +1,49 @@
+package processor
+
+import (
+	"context"
+
+	"github.com/go-kit/kit/endpoint"
+	//"github.com/go-kit/kit/log"
+)
+
+type Endpoints struct {
+	CalculateEndpoint endpoint.Endpoint
+}
+
+func (e Endpoints) Calculate(ctx context.Context, value int, multiplier int) (int, error) {
+	resp, err := e.CalculateEndpoint(ctx, CalculateRequest{Value: value, Multiplier: multiplier})
+	if err != nil {
+		return 0, err
+	}
+	response := resp.(CalculateResponse)
```

Comment:
handling the type assertion in this way can be finished in an error, consider using the:
```go
  response, ok := resp.(CalculateResponse)
  if ok != nil {
      return 0, fmt.Errorf("error ......")
  }
  // rest of code...
```